### DISCUSSION TITLE
Clozure CL Version 1.13  & Maxima

### DISCUSSION THREAD

**Discussion by barton-willis** [Clozure/ccl#516] [General]
Clozure CL Version 1.13 builds the Maxima CAS OK using Windows. It runs the Maxima testsuite with one failure that involves floating point accuracy. The failure is a relative error of 8.09e-15 when the upper bound is 8.0e-15 for an evaluation of an incomplete gamma function. Such accuracy failures are not uncommon--last I tried the newest SBCL has several similar failures.  

Thanks for the new Clozure version.

--Barton
   Maxima developer

**Comment by xrme** [comment]
Thank you for that positive report.

**Comment by barton-willis** [comment]
The new version resolved a few bugs in the Maxima code that outputs floating point numbers in engineering format. These tests are in Maxima's share testsuite and I didn't run them yesterday.

**Comment by xrme** [comment]
Although a 64-bit Lisp is probably a better option for Maxima in nearly all circumstances, I thought I would mention that I just fixed the 32-bit x86 CCL (see #377) so that it can at least build Maxima now.

The fix isn't in the 1.13 release, so it's necessary to rebuild from source to pick up the fix.
